src/js-v2-api-specification.md

```diff
@@ -0,0 +1,309 @@
+
+#### Parameters
+
+| Parameters              |                                                                                                                                                                |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| subject                 | **Type:** string *Required*<br>The Subject line for the calendar appointment.                                                                                                            |
+| body                    | **Type:** string *Required*<br>Plain text body for the appointment.                                                                                                                      |
+| location                | **Type:** string *Required*<br>Plain text location for the appointment.                                                                                                                  |
+| startTimeEpochInSeconds | **Type:** number *Required*<br>Start datetime for the appointment, set as a Epoch timestamp.                                                                                             |
+| endTimeEpochInSeconds   | **Type:** number *Optional*<br>End datetime for the appointment, set as a Epoch timestamp. Must be greater or equal to `startTimeEpochInSeconds`.  Defaults to the same value as the start time (matching default behaviour of iOS and Android). |
+
+##### Example Usage
+
+```js
+nhsapp.storage.addEventToCalendar(
+	"Hospital Appointment",
+	"Please show up",
+	"Dr Surgery, LS1",
+	1592211600,
+	1592215200
+)
+```
+
+##### Status
+
+Live
+
+---
+
+#### downloadFromBytes Function <a name="downloadFromBytes"></a>
+
+```	javascript
+nhsapp.downloadFromBytes(
+  base64data: string,
+  filename: string,
+  mimeType: string
+)
+```
+
+##### Description
+
+Download a file to the user's device. The NHS App's native controls will prompt the user to select a location.
+
+The mime type `application/vnd.apple.pkpass` is supported on the iOS platform and can be used to add a pass into the iOS wallet. The [getAppPlatform](/nhsapp-developer-documentation/js-api-specification#getAppPlatform) api can be used to determine the current application platform.
+
+#### Parameters
+
+| Parameters              |                                                                                                                                                                |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| base64data                 | **Type:** string *Required*<br>A base64 string representation of the file.                                                                                                            |
```

Comment:
Not sure how much it matters, but whitespace alignment.

Reply:
fixed